libcaf_core/caf/action.hpp

```diff
@@ -149,6 +149,8 @@ struct default_action_impl : detail::atomic_ref_counted, action::impl {
     // to implement time-based loops.
```

Comment:
That comment is not accurate anymore. There is not a path that changes the state to disposed.